### DISCUSSION TITLE
Override day flash

### DISCUSSION THREAD

**Discussion by MaxmHz123** [maca88/SmartBikeLights#78] [General]
Hi Maca88, thank you for your SmartBikeLights data field. Great work. Unfortunately my configuration is not working. I want to run my Varia 515 in day flash mode by default. When a car approaches I want it to go into solid mode. On my SmartBikeLights overview on the Edge it also shows me the switch to solid mode. However, the Varia does not make this change and increases the flashing frequency due to the approaching car. Is it a configuration error on my part or is it not possible to switch out of day flash mode when there is an approaching vehicle, as Varia internal logics are interfering with this?

`###0,73535488::1#2,2!F2S:1:4:0:0I[150]0!:1:7:0:0D=1##6,5:Varia 515:0:16777215!2,:-1,Off:0!1,Solid:4!1,Peloton:5!1,Day Flash:7!1,Night Flash:6#0::#0:1#123!:123!#0#0#B2713##3#0#0`

br
Max

**Comment by maca88** [comment]
Hi,

> When a car approaches I want it to go into solid mode

Unfortunately that is not possible. Varia has a non configurable flash behavior when a vehicle approaches:
https://www8.garmin.com/manuals/webhelp/GUID-C41F445D-457F-447D-88C8-FE286BF157E9/EN-US/GUID-4587F7D3-DEE0-422D-9450-D125E15C4EA0.html

With a CIQ application (e.g. Smart Bike Lights), you can only set the light mode (Mode column from the above link) which will use a predefined flash mode when a vehicle will approach you. So, you can only choose between various flash behaviors (e.g. Peleton -> flashes softly)

**Comment by MaxmHz123** [comment]
Thank you very much. I changed it to peleton and now I have almost the desired behavior.
